build.ts

```diff
@@ -0,0 +1,172 @@
+import { build as esbuild, BuildOptions } from 'esbuild';
+import util from 'util';
+import chalk from 'chalk';
+import { promises as fs } from 'fs';
+import { watch } from 'chokidar';
+import { debounce } from 'debounce';
+import origRimraf from 'rimraf';
+import child_process from 'child_process';
+
+const rimraf = util.promisify(origRimraf);
+const exec = util.promisify(child_process.exec);
+
+/**
+ * @todo
+ *
+ * - Figure out how to export the fonts and include them with dynamic urls in the css?
```

Comment:
These fonts were unused by our web reader but included in the ReadiumCSS. Not sure how they were dealt with before, possibly just uncaught errors that we didn't know about. Anyways, shouldn't be a problem for now